views/pulsar/v2/helpers/elem.html.twig

```diff
@@ -43,11 +43,29 @@ value       | string | Specifies the value of the input
         {% set options = options|defaults({'class': 'form__control'}) %}
     {% endif %}
 
+    {# Build aria-described string of guids for help blocks and errors #}
+    {% set describedByIds = [] %}
+
     {% if options.help_id is defined and options.help_id is not empty %}
-        {% set options = options|defaults({'aria-describedby': options.help_id}) %}
+        {% set describedByIds = describedByIds|merge([options.help_id])%}
+    {% endif %}
+
+    {% if options.error_ids is defined and options.error_ids is not empty %}
```

Comment:
Is `error_ids` an internal-only value passed between macros or is it user-supplied? Can't see any reference to it

Reply:
It's internal-only. `error_ids` is created in the extension, which `group` calls to modify `options`, which then gets passed down to the macros in `elem.html.twig`.